scripts/generate-changed-recipes

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
```suggestion
#!/usr/bin/env python
```

Add `.py` to script name

Reply:
0954429b463c5276be7898e733b00a87dac3b2ca